tools/provision-vs2015.ps1

```diff
@@ -0,0 +1,507 @@
+#  Copyright (c) 2014-present, Facebook, Inc.
```

Comment:
This is a _massive_ amount of code duplication. What components are you needing out of this? We should take whatever functionality is needed here, and fold this into the generic `provision.ps1`, _or_ dot source whatever functionality is needed from `provision.ps1` and keep all of our provisioning logic there. In this case I'd say let's stick with the former.